sources/packages/api/src/utilities/datatable-config.ts

```diff
@@ -10,3 +10,12 @@ export enum FieldSortOrder {
  */
 export const DEFAULT_PAGE_LIMIT = 10;
 export const DEFAULT_PAGE_NUMBER = 0;
+
+// pagination option
+export class PaginationOptions {
```

Comment:
Can you please covert it to an interface if there is no reason to it be a class?